### ISSUE TITLE
Odd font for capital K

### ISSUE THREAD

**Issue by Isabetta** [uckelman/dw_op#39]
Several people whos names begin with K have an odd font for the capital K where it has a white outline. Morten Kanik is an example of this but I have seen it in many places

**Comment by uckelman** [comment]
I don't see this when I look up Morten Kanik. Is this still happening for you?

**Comment by Isabetta** [comment]
Yes I still see the odd font for K. I am using chrome as my browser on my
computer. See pictures of his site and a couple of other random people
whose names starts with K. this email bounced back cause the size of
pictures was too big. I will send the pictures to you using facebook
mesenger.

I tried searching using Edge as a browser and then the K looks normal so it
is something wierd with my browser. Using chrome in my android phone also
looks normal. I will try to remember to check at work tonight to see what
happens on another browser.

It is no big deal so don't spend a lot of time trying to figure this out.

Isabetta
